packages/app/src/cli/services/dev/processes/setup-dev-processes.test.ts

```diff
@@ -26,12 +26,15 @@ import {describe, test, expect, beforeEach, vi} from 'vitest'
 import {ensureAuthenticatedAdmin, ensureAuthenticatedStorefront} from '@shopify/cli-kit/node/session'
 import {Config} from '@oclif/core'
 import {getEnvironmentVariables} from '@shopify/cli-kit/node/environment'
+import {isStorefrontPasswordProtected} from '@shopify/theme'
```

Comment:
Updated these tests to assert that the process list that is built uses the `TS` implementation coming from `theme-app-extension-next` rather than the legacy implementation